FuzzTesting/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.10
```

Comment:
I'm guessing because of this change you need to swap some things to `executableTarget`.  See the output of the failing CI actions for the warning that I'm basing this on.

@gjcairo for any other ideas for the failing fuzz tests.

Reply:
Another random guess - What if you edit `FuzzTesting/do_build.sh` to remove the pass through of `-parse-as-library`?

Reply:
Thanks for the suggestion. I’m going to look in the morning which Swift versions reproduce this.

Reply:
I'm still trying things out but it looks like there are issues with fuzzing and SwiftPM
https://github.com/swiftlang/swift/issues/75800
https://github.com/swiftlang/swift-package-manager/issues/4522
https://github.com/swiftlang/swift-package-manager/issues/4769

Reply:
Lowering the tools version back to 5.3 seems to have resolved the immediate issue.

Reply:
@thomasvl Are you happy with this approach?